### PR TITLE
Add Control.Lens.Unsound

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: src/Control/Lens/Unsound.hs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+#if __GLASGOW_HASKELL__ < 708
+{-# LANGUAGE Trustworthy #-}
+#endif
+
+{-# LANGUAGE RankNTypes #-}
+
+-------------------------------------------------------------------------------
+-- |
+-- Module      :  Control.Lens.Unsafe
```
Comment (Taneb) [inline src/Control/Lens/Unsound.hs]: I'm like 995 sure this should be Control.Lens.Unsound

Comment (phadej) [reply src/Control/Lens/Unsound.hs]: fixed

**Comment by phadej** [comment]
finally green.

**Comment by phadej** [comment]
@ekmett any comments on this, should I merge?

**Comment by ekmett** [comment]
Let's start here and keep dumping things in there. There are similar products for things like traversals, etc.

**Pull request merged**